src/app/pages/quickMaps/quickMaps.service.ts

```diff
@@ -21,6 +21,10 @@ export class QuickMapsService {
   // eslint-disable-next-line @typescript-eslint/member-ordering
   public popGroupIdObs = this.popGroupIdSrc.asObservable();
 
+  private readonly mndDataIdSrc = new BehaviorSubject<string>(null);
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
this rule keeps getting pinged. If we are regularly needing to use a private var within a public var, it might be a rule to consider removing https://github.com/micronutrientsupport/micronutrient-support-tool/issues/87